src/aria2p/cli.py

```diff
@@ -466,11 +484,20 @@ def subcommand_add(api: API, uris: List[str] = None, from_file: str = None) -> i
         from_file: Path to the file to read uris from.
             Deprecated: Every URI that is a valid file-path
             and is not a torrent or a metalink is now read as an input file.
+        options: String of aria2c options to add to download.
+        position: Position to add new download in the queue.
 
     Returns:
         int: 0 if OK else 1.
     """
     uris = uris or []
+    position = position or "0"
+
+    if options:
+        options = {
+            str(opt).strip(): str(val).strip()
+            for opt, val in (download_option.split("=", 1) for download_option in options.split(",", 1))
```

Comment:
We should make sure no option value can contain commas `,`! We'll have to read the man page :sweat_smile:
Or our own docs https://pawamoy.github.io/aria2p/reference/options/ :stuck_out_tongue:

Reply:
Oh, maybe we could write a function that does this instead, to use it as `type` in the argparse option. This way we wouldn't have to copy-paste if we add these options to the other `add` CLI commands.